bigquery/Dockerfile

```diff
@@ -1,7 +1,7 @@
 # When running this Dockerfile context must be project root, in order to pick
 # up indexer_util and dataset_config
 
-FROM python:2
+FROM python:3
```

Comment:
Please make branch in this repo, so circleci will run.

Reply:
I will create a new PR from a branch in this repo.